data/remnant missions.txt

```diff
@@ -60,27 +62,94 @@ mission "First Contact: Remnant"
+			`	"Unfortunately you have discovered our location, and we depend greatly on secrecy to maintain our existence. We are willing to let you work and live among us if you and your crew will submit to a genetic test, or you can leave if you commit to never sharing what you have learned with the rest of the galaxy. The choice is yours."`
+			choice
+				`"Oh, that makes sense."`
+					goto ReturnConsent
+				`"I'm going to forget you exist."`
+					goto LeaveForever
+			label ReturnConsent		
+			apply
+				"reputation: Remnant" = 1
+			`	"We will let them know to expect you, but be on your best behavior, as there will be no third chance." The captain pauses for a moment, then continues "Moreover, the blood test is mandatory for you and your crew. We will not tolerate anyone of any Alpha heritage on our worlds."`
+				goto SCEnd
+			label LeaveForever
+			apply
+				set "Remnant Second Chance Refused"
+			`	"Please do. You won't be welcome if you return.`
+				flee
+			label SCEnd
+			
+	on complete
+		fail "First Contact: Remnant"
+		conversation
+			`For a second time you descend through the atmosphere of Viminal. Unlike the first time, your flight computer keeps blaring warnings that the ships escorting you are weapons-hot and locked on. As soon as you land your ship is surrounded by soldiers with a variety of weapons, and the same man appears in the midst of them. As before, he pricks his finger, presses it to a spot on the device, then offers a still-sealed lancet to you. "The first may be a misunderstanding, but now you know our law. Will you submit to a blood test, or be declared guilty once and for all?" he chants in a severe tone.`
+			choice
+				`(Provide the blood sample.)`
+					goto SecondChanceComply
+				`(Refuse and run for your ship.)`
+					goto SecondChanceRefuse
+			label SecondChanceRefuse
+			`	As you turn to start running back to your ship you notice a quick hand sign from the prefect and the soldiers nearby begin raising their weapons and firing at you. As you feel your body begin to liquify under the assault of the strange energy bolts, the last thing you see is the look of pity, resignation, and confusion on the face of the Prefect. Probably wondering why you would decide to return if you were just going to run again. You suppose they'll never know...`
+			die
+			label SecondChanceComply
+			apply
+				"Remnant Blood Test Pure" = 1
+			`	Several of them gather around the one with the scanner as if waiting to see what the results will be. As you watch them you notice that all of them, even the women, have their hair cropped short. Also, their clothing looks like nothing that has ever been in fashion at any point in human history that you know of.`
+			`	Finally the scanner beeps, and a yellow light flashes. You can almost feel the tension leaving the air as the crowd relaxes. "So you are pure. You are human," chants one of the women. "You are one of us."`
```

Comment:
Hmmm... I don't read it as "Happy-go-lucky," but most of that text is the same as the original first contact mission, and is aimed at ensuring that by the time the player is accepted by the Remnant they have the same baseline information either way. That being said, I rather like the transition because it honestly is going from "They are about to shoot you" to "They are happy you are one of them."

Reply:
I don't read this as happy-go-lucky, they were hostile due to suspicion, and they obviously trust their scanner, so there's no reason for their hostility to carry over